doc/19-technical-concepts.md

```diff
@@ -632,16 +632,19 @@ The algorithm works like this:
  * Set the authority (true or false)
 
 The object authority calculation works "offline" without any message exchange.
-Each instance alculates the SDBM hash of the config object name, puts that in contrast
-modulo the connected endpoints size.
+Each instance calculates the SDBM hash of the config object name up to the
+host name separator "!", puts that in contrast modulo the connected endpoints size.
 This index is used to lookup the corresponding endpoint in the connected endpoints array,
 including the local endpoint. Whether the local endpoint is equal to the selected endpoint,
 or not, this sets the authority to `true` or `false`.
 
 ```cpp
-authority = endpoints[Utility::SDBM(object->GetName()) % endpoints.size()] == my_endpoint;
+authority = endpoints[Utility::SDBM(object->GetName().SubStr(0, object->GetName().FindFirstOf('!'))) % endpoints.size()] == my_endpoint;
```

Comment:
Does it even make to include that implementation specific C++ code snippet here? Which extra information does it provide, but confusing the user? I would completely remove that line from here, after all, the description above it is there to describe what this c++ code does.

Reply:
It was already there, I've just updated it.

Also, I consider it a good in-a-nutshell summary of the (more verbose) text.